src/main/java/marquez/api/models/NamespaceResponse.java

```diff
@@ -30,7 +30,7 @@
   @JsonProperty("createdAt")
   String createdAt;
 
-  @JsonProperty("owner")
+  @JsonProperty("connectionUrl")
```

Comment:
Did you mean to rename this field?

Reply:
Will be fixed in the next revision.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.